site/content/recipes/perform_code_review.md

```diff
@@ -63,36 +63,56 @@ use it or are used by it.
 
 ## Reviewer reviews code
 
-What should you do if you're the reviewer, providing feedback on someone else's
-code?
-We suggest a workflow with two key elements.
+Code review can seem intimidating, even for the reviewer. 
+Where do you even start? What do you look at? 
+The good news is: there are a number of points that are almost always worth looking at. 
+In particular, the readability is often a good place from which to start a code review. 
```

Comment:
Is `the raedability` correct here? Or just `readability`?

Reply:
```suggestion
In particular, readability is often a good place from which to start a code review. 
```